daisy_workflows/image_build/debian/build.py

```diff
@@ -102,9 +102,8 @@ def main():
   # Upload tar.
   image_tar_gz = '/target/disk.tar.gz'
   if os.path.exists(image_tar_gz):
-    image_tar_gz_dest = os.path.join(image_dest, 'root.tar.gz')
-    utils.Status('Saving %s to %s' % (image_tar_gz, image_tar_gz_dest))
-    utils.Gsutil(['cp', image_tar_gz, image_tar_gz_dest])
+    utils.Status('Saving %s to %s' % (image_tar_gz, image_dest))
```

Comment:
Nit: prefer this:

    'Saving {} to {}'.format(image_tar_gz, image_dest)

to this:

    'Saving %s to %s' % (image_tar_gz, image_dest)

because it's Python 3-compatible.